src/Backupper/AbstractBackupper.php

```diff
@@ -138,9 +139,21 @@ public function isVerbose(): bool
         return $this->verbose;
     }
 
+    public function setTimeout(float $timeout): self
```

Comment:
Why float if the original value is an integer ?

Reply:
I chose float because the symfony process we set the timeout for also uses a float

Reply:
Oh, I see, I don't have a strong opinion then, but I'd still prefer to keep int and keep seconds as the lowest unit. It's backup and restore operations and they will probably never run under a second. What's your opinion about this @SimonMellerin ?

Reply:
Ok I understand you choice but perhaps we can deviate from the behavior of Symfony Process Component here. 

The Symfony Process Component addresses general purpose  process than can be speeder than a second. But as @pounard said, our use case is a more specific and we will never need to be so precise with this timeout.

This being said, I found it easier to understand to have an integer for this configuration.